js/src/views/Dapps/Summary/summary.js

```diff
@@ -38,7 +38,14 @@ export default class Summary extends Component {
       return null;
     }
 
-    const url = `/app/${app.builtin ? 'global' : 'local'}/${app.url || app.id}`;
+    let type = 'builtin';
```

Comment:
Fixed. Apps in installed in `.pariyt/dapps`, builtins & network apps now work as expected.